nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -112,38 +113,67 @@ let
       in mkOption {
         type = types.enum ((attrValues available) ++ (attrNames available));
         apply = x: if isInt x then x else available.${x};
-        description = ''
+        description = lib.mdDoc ''
           Permission level to restrict the actor(s) to.
           See <https://www.dokuwiki.org/acl#background_info> for explanation
         '';
         example = "read";
       };
-
     };
   };
 
-  siteOpts = { config, lib, name, ... }:
+  # Lite re-implementation of mkRenamedOptionModule and doRename so that we can show the full path
```

Comment:
Any reason to _not_ merge this into the primary implementation of `mkRenamedOptionModule`?

Reply:
It should be more explicit on the comment. Something like "the functionality X is not provided by the current implementation of Y".